src/pages/patientView/PatientViewPage.tsx

```diff
@@ -540,6 +546,17 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </MSKTab>
                     )}
 
+                    {(getBrowserWindow().localStorage.trialmatch === 'true') && (
+                        <MSKTab key={7} id="trialMatchTab" linkText="Trial Match"
+                                hide={!this.shouldShowTrialMatch(patientViewPageStore)}>
+                            <TrialMatchTable sampleManager={sampleManager}
+                                             trialMatches={patientViewPageStore.trialMatches.result}
+                                             trials={patientViewPageStore.matchMinerTrials.result}
+                                             nctTrials={patientViewPageStore.nctTrials.result}
+                                             containerWidth={WindowStore.size.width-20}/>
```

Comment:
I don't think you really need to set the table width. Just set a width for ID and intervention column and let rest two columns adjust to the view should be good enough.

Reply:
No, I need to set width otherwise title column will occupy so much space.